clients/java/signalr/src/main/java/JsonHubProtocol.java

```diff
@@ -46,35 +46,34 @@ public TransferFormat getTransferFormat() {
                     //Invocation Message
                     String target = jsonMessage.get("target").getAsString();
                     JsonElement args = jsonMessage.get("arguments");
-                    invocationMessages.add(new InvocationMessage(target, new Object[] {args}));
+                    hubMessages.add(new InvocationMessage(target, new Object[] {args}));
+
                     break;
                 case "2":
```

Comment:
Can we use the enum values now?

Reply:
So here in the HubProtocol level I just pull the value right from the JSON and construct the specific HubMessage type but in a list of the base type `HubMessage`. 
The enum values are used in the HubConnection layer when we actually need to cast to the specific message type and need to decide what to do with each message of each type